README.md

```diff
@@ -52,6 +52,7 @@ Here's a list of supported commands:
   set-replication-factor [topic] [--newrf <n>] [--brokers id[,id]]    Set the replication factor of
   shutdown <broker id>                                                Gracefully remove leaderships from a broker (requires JMX).
   topics                                                              Print all topics.
+  drain <broker id> [topic] [--brokers <ids>]                         Reassign partitions from a dead broker to healthy brokers.
```

Comment:
`dead broker` is a bit pessimistic ;)
I'd simply say "Reassign the partitions owned by a specific broker"